src/color/hslToRgb.js

```diff
@@ -1,21 +1,21 @@
 const hueToColorComponent = require('./hueToColorComponent')
 
 /**
- * Converts an HSL color value to RGB. Conversion formula
- * adapted from http://en.wikipedia.org/wiki/HSL_color_space.
+ * Converts an HSL color value to RGB.
  * Assumes h, s, and l are contained in the set [0, 1] and
  * returns r, g, and b in the set [0, 1].
  *
+ * @see http://en.wikipedia.org/wiki/HSL_color_space.
  * @param   Number  h       The hue
  * @param   Number  s       The saturation
  * @param   Number  l       The lightness
  * @return  Array           The RGB representation
  */
 const hslToRgb = (h, s, l) => {
```

Comment:
same remark as above, perhaps h, s, l , a to get a toRgba ?

Reply:
Changed to accept an array of HSL or HSLA values.